CMakeLists.txt

```diff
@@ -290,10 +290,10 @@ endif()
 set (FILAMENT_SAMPLES_BINARY_DIR ${PROJECT_BINARY_DIR}/samples)
 
 if (WEBGL)
-    add_subdirectory(${FILAMENT}/samples/web)
+    add_subdirectory(web/filamentjs)
```

Comment:
to rename to `filament-js` to follow the Android convention

Reply:
Fixed.